tests/test_type/test_tuple.py

```diff
@@ -258,9 +256,7 @@ class MyCircuit(m.Circuit):
     def DefineTop(params):
         class Top(m.Circuit):
             io = m.IO(IFC1=IFC1(params))
-            @classmethod
-            def definition(io):
-                m.wire(io.IFC1.port4, DefineMyCircuit(params)().IFC0.port4)
+            m.wire(io.IFC1.port4, DefineMyCircuit(params)().IFC0.port4)
         return Top
 
     m.compile("top", DefineTop({'param0': 5}))
```

Comment:
It looks like these tests don't really check that things work, which is why #609 was probably not caught by travis (still not sure why the CoreIR exit code is not set properly on travis vs. in #609...)

Reply:
Hmm yea that's strange, we should probably check the exit code inside compile and raise an exception (if we don't already)

Reply:
We do have that check in the code: https://github.com/phanrahan/magma/blob/5bdb58bda377b363824629b719ddb0a4fb433fa8/magma/backend/coreir_compiler.py#L55-L56

but for some reason we're not hitting it on travis...